### PR TITLE
SamyGO.sms4tv version 1.03

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109539&drop=dogfoodAlpha

### CODE REVIEW

**Review by ItzLevvie — APPROVED** [review]
```
C:\Users\TAP>sms4tv
SMS4TV 1.03: freeware command-line utility for Samsung TV-set. (c)2010 geo650

This tool can send short message to your TV-set using network connection.

usage: sms <to> <message> <from> [host]
example:
sms "My Dear" "Hello! I'll be back at 7pm. See you." George 192.168.1.5

Credits: thanks to Ewoud Werkman and pter-s for their java applications
and thanks to all the people from SamyGO forum.
```

**Review by ItzLevvie — APPROVED** [review]

**Comment by upintheairsheep** [comment]
@ItsLevvie make sure to add the copyright to the manifest prior to submitting

File: manifests/s/SamyGO/sms4tv/1.03/SamyGO.sms4tv.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: SamyGO.sms4tv
+PackageVersion: 1.03
+PackageLocale: en-US
+Publisher: SamyGO
+PackageName: sms4tv
+License: Proprietary
+ShortDescription: This tool can send short message to your TV-set using network connection.
```
Comment (stephengillie) [inline manifests/s/SamyGO/sms4tv/1.03/SamyGO.sms4tv.locale.en-US.yaml:10]:
```suggestion
ShortDescription: This tool can send a short message to your TV using a network connection.

```

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
@upintheairsheep: You need to commit the changes from https://github.com/microsoft/winget-pkgs/pull/109539/commits/586f7e774128ee7fb397f5a5bcf7ca20de4f7e1d#r1227385369

**Review by stephengillie — APPROVED** [review]

**Comment by japplis** [comment]
This merge causes some problems that makes it difficult to create new MR. The reason is that there is a SamyGo and a SamyGO company. On Windows file system, it's the same directory and in github 2 directories.
I would suggest to revert this commit and contact the commiter to commit in the correct SamyGo directory.

**Pull request merged**